cmake/all.cmake

```diff
@@ -57,10 +57,11 @@ set(CMAKE_PREFIX_PATH_AS_IS ${CMAKE_PREFIX_PATH})
 # list of unique catkin workspaces based on CMAKE_PREFIX_PATH
 set(CATKIN_WORKSPACES "")
 foreach(path ${CMAKE_PREFIX_PATH})
-  if(EXISTS "${path}/.catkin")
-    list(FIND CATKIN_WORKSPACES ${path} _index)
+  file(TO_CMAKE_PATH "${path}" cmake_path)
+  if(EXISTS "${cmake_path}/.catkin")
```

Comment:
Please keep this condition as is and instead move the `file(TO_CMAKE_PATH ...)` line into the if block since it is only used there.